### PR TITLE
fix(IAM Policy Management): add restore policy functionality

### PR DESCRIPTION
**Pull request by swcolley**
## PR summary

Updating SDK to include new changes from API for PATCH for v1/policies and additional new params.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/iam-policy-management.v1.test.js [hunk 1]
```diff
@@ -292,7 +292,37 @@ describe('IamPolicyManagementV1', () => {
       });
 
     // end-delete_policy
-  })
+  });
+  test('patchPolicy request example', done => {
```
Comment (guicabr) [inline examples/iam-policy-management.v1.test.js:296]: I think you might need to move the PATCH example/test after UPDATE like you did in Go and Python case

Comment (swcolley) [reply examples/iam-policy-management.v1.test.js:296]: Good catch.

File: examples/iam-policy-management.v1.test.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ const originalWarn = console.warn;
 const consoleLogMock = jest.spyOn(console, 'log');
 const consoleWarnMock = jest.spyOn(console, 'warn');
 
-describe('IamPolicyManagementV1', () => {
+describe.only('IamPolicyManagementV1', () => {
```
Comment (padamstx) [inline examples/iam-policy-management.v1.test.js:53]: I'm not familiar with `.only` in this context.  Just curious what the significance is here.

**Review by padamstx — APPROVED** [review]
LGTM
Just wondering if we really need the .only() in the examples is all.

File: examples/iam-policy-management.v1.test.js (hunk 2 above)
Comment (swcolley) [reply examples/iam-policy-management.v1.test.js]: .only is used by different frameworks to run specific tests (mocha-chai), I left it in before realizing jest does not use .only after the describe and missed removing it.

**Pull request merged**